tools/clone_flutter.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -e
+set -x
+
+if [ -z $ENGINE_PATH ]
```

Comment:
Surround the path with quotes, otherwise this won't work when the variable is unset.

```sh
if [ -z "$ENGINE_PATH" ]
```

Reply:
See also https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/base/bot_detector.dart

Reply:
@tvolkert done.
@yjbanov thanks for sharing. I am testing $CIRRUS_CI env parameter now. If it works I'll also add this as a check.

Reply:
thanks for the suggestion. I also add a check for the env. flag.